main.go

```diff
@@ -35,11 +35,10 @@ import (
 	"time"
 )
 
-const volMount = "/git"
-
 var flRepo = flag.String("repo", envString("GIT_SYNC_REPO", ""), "git repo url")
 var flBranch = flag.String("branch", envString("GIT_SYNC_BRANCH", "master"), "git branch")
 var flRev = flag.String("rev", envString("GIT_SYNC_REV", "HEAD"), "git rev")
+var flRoot = flag.String("root", envString("GIT_SYNC_ROOT", "/git"), "root directory for git operations")
```

Comment:
This seems like an implementation detail. Is this actually useful to parametrize?